src/shared/api/urls.ts

```diff
@@ -219,9 +219,11 @@ export function getDocsUrl(sourceUrl:string,docsBaseUrl?:string): string {
 }
 
 export function getWholeSlideViewerUrl(ids: string[], userName: string): string {
-    return ids.length > 1 ? `https://slides.mskcc.org/cbioportal?ids=${ids.join('.svs;') + '.svs'}&user=${userName}&annotation=off` : ids.length === 1 ? `https://slides.mskcc.org/cbioportal?ids=${ids[0] + '.svs'}&user=${userName}&annotation=off&filetree=off` : "";
+    const tokenInfo = JSON.parse(AppConfig.serverConfig.mskWholeSlideViewerToken);
```

Comment:
you should always wrap a parsing operation in try{} catch {} in case the input is invalid.

Reply:
Yes! Thanks for the reminder.